### DISCUSSION TITLE
DDC min value don’t match iPlaoe display

### DISCUSSION THREAD

**Discussion by Jie2GG** [MonitorControl/MonitorControl#934] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

The brightness adjusted by the software does not match the my display brightness.
I tried changing the DDC minimum to get a consistent tuning range with the display. 
But when I tried to adjust the slider to 0, the brightness of the monitor was 22. 
I tried to change the value to negative and found that this operation is not supported.

### Expected behavior

I would like a solution to the problem.

### Anything else?

None

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.2 Monterey
- Mac model: Hackintosh
- MonitorControl version: v4.0.2
- Monitor(s): iPlaoe
- Monitor Cable(s)/Connection(s): Notebook -> HDMI -> iPlaoe
- Apple Silicon/M1 (yes or no): no
```

**Comment by waydabber** [comment]
This seems to be an issue with the display firmware. You might want to try m1ddc and see if adjusting the brightness has the same results. You can check your display's menu settings, maybe there is something that causes this behavior.
